### PR TITLE
LTD-5452-feedback-content-change

### PR DESCRIPTION
**Pull request by depsiatwal**
https://uktrade.atlassian.net/browse/LTD-5452

New feedback form where the exported get a link to the support-help desk link. 
Had to refactor and create an exporter specific feedback form. 

Also took this opportunity to use our standard base crisp_form.

### CODE REVIEW

File: exporter/core/feedback/forms.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from crispy_forms_gds.layout import HTML
+
+from core.feedback.forms import FeedbackForm
+
+
+class ExporterFeedbackForm(FeedbackForm):
```
Comment (saruniitr) [inline exporter/core/feedback/forms.py:6]: `FeedbackForm` alone used separately? just wondering why it is split like this.

File: unit_tests/exporter/feedback/test_views.py [hunk 2]
```diff
@@ -14,6 +15,16 @@ def test_feedback_get_success(url, authorized_client):
...
 
 
+def test_feedback_contains_support_link(authorized_client):
+    url = reverse("feedback")
+    response = authorized_client.get(url)
+    assert response.status_code == 200
+    soup = BeautifulSoup(response.content, "html.parser")
+    links = soup.find_all(class_="govuk-link")
+    support_link = [l["href"] for l in links if l["href"] == "/help-support/"]
+    assert support_link
```
Comment (saruniitr) [inline unit_tests/exporter/feedback/test_views.py:25]: Could you also add a test when this form is submitted?

**Review by hnryjmes — COMMENTED** [review]
when trying to submit the form with some text in, even as a logged in user, I get this error: 
```
403 - [{'error': 'AuthError', 'message': 'Invalid token: service id is not the right data type'}]
```

possibly this is me doing something wrong, have you seen this error before?

File: core/feedback/forms.py [hunk 3]
```diff
@@ -1,5 +1,19 @@
 from django import forms
+from crispy_forms_gds.layout import HTML
 
+from core.common.forms import BaseForm
 
-class FeedbackForm(forms.Form):
-    feedback = forms.CharField(label="Please type your feedback here: ", widget=forms.Textarea)
+
+class FeedbackForm(BaseForm):
```
Comment (hnryjmes) [inline core/feedback/forms.py:7]: good use of BaseForm

File: core/feedback/forms.py [hunk 4]
```diff
@@ -1,5 +1,19 @@
...
+    class Layout:
+        TITLE = "Leave Feedback"
```
[the remark this answers is not in the document]
Comment (depsiatwal) [reply core/feedback/forms.py]: done

File: exporter/core/feedback/forms.py (hunk 1 above)
Comment (depsiatwal) [reply exporter/core/feedback/forms.py:6]: prior to this FeedbackForm was used by exporter/caseworker 

No we have a requirement to change some the content on exporter feedback hence why I created a specialised version for it rather then creating completely separate views/forms with lots of overlapping functionality

File: unit_tests/exporter/feedback/test_views.py (hunk 2 above)
Comment (depsiatwal) [reply unit_tests/exporter/feedback/test_views.py:25]: This is the text below 
test_feedback_post_success

**Comment by depsiatwal** [comment]
> when trying to submit the form with some text in, even as a logged in user, I get this error:
> 
> ```
> 403 - [{'error': 'AuthError', 'message': 'Invalid token: service id is not the right data type'}]
> ```
> 
> possibly this is me doing something wrong, have you seen this error before?

Could be quite possible you don't have email/ notify api not setup locally

**Review by markj0hnst0n — APPROVED** [review]
Setup notify locally and all seems to be working well.  I get that the support link is the main change here so test makes sense.

**Pull request merged**